hie-bios.cabal

```diff
@@ -1,21 +1,27 @@
+Cabal-Version:          2.4
```

Comment:
I might be wrong, but doesnt this force us to build this package with cabal 2.4 at least? 
This will break old stack lts snapshots, since they got shipped with older cabal versions. Iirc, it would force hie to drop stack support for GHC versions older than `8.6`. 
At least lower the `Cabal-Version` to 2.2. What was the problem with `1.1`?

Reply:
You need `Cabal-2.4` for `tests/projects/**/*.hs` syntax.